samples/app_event_manager/sample.yaml

```diff
@@ -59,12 +59,12 @@ tests:
     extra_configs:
       - CONFIG_SHELL=y
     platform_exclude: native_posix
-    tags: sysbuild
+    tags: sysbuild app_event_manager
```

Comment:
Should we also tag `tests/subsys/app_event_manager`? Should we tag applications that use Event Manager?

Reply:
tests/subsys/app_event_manager - good point, I will add.

> Should we tag applications that use Event Manager?
No, the approach is different.
Tags are used to exclude testing not to find something to be tested.

Reply:
Discussed offline and aligned approach.